webClient/src/app/app.module.ts

```diff
@@ -11,14 +11,37 @@
 */
 
 import { CommonModule } from '@angular/common';
-import { NgModule } from '@angular/core';
+import { NgModule, Inject } from '@angular/core';
 import { FormsModule } from '@angular/forms';
 
 import { ZluxButtonModule, ZluxPopupManagerModule } from '@zlux/widgets';
 
 import { AppComponent } from './app.component';
 import {HelloService} from './services/hello.service';
 
+// import { L10nConfig, L10nLoader, TranslationModule, StorageStrategy, ProviderType } from 'angular-l10n';
+import { TranslationModule, L10nConfig, ISOCode, L10nLoader, LOCALE_CONFIG,
+  TRANSLATION_CONFIG, LocaleConfig, TranslationConfig } from 'angular-l10n';
+import { Angular2L10nConfig, Angular2InjectionTokens } from 'pluginlib/inject-resources';
+
+
+const l10nConfig: L10nConfig = {
+  locale: {
+      languages: [
+          { code: 'en', dir: 'ltr' },
+          { code: 'ja', dir: 'ltr' }
```

Comment:
This seems like it is missing languages. What is the effect?

Reply:
No bearing on the output, will be removed